tests/utils/test_url.py

```diff
@@ -18,6 +18,17 @@ def test_join_url_fragments(base: str, fragment: str, expected: str) -> None:
     assert join_paths([base, fragment]) == expected
 
 
-@pytest.mark.parametrize("base,expected", [("/path", "/path"), ("path/", "/path"), ("path", "/path")])
+@pytest.mark.parametrize(
+    "base,expected",
+    [
+        ("", ""),
```

Comment:
Shouldn't this be `/`?

Reply:
I would have thought so, but quite a few tests break if it is.

Reply:
Fixed by changing the `join_paths` function to always return normalized paths